configuration/analyzer_config.json

```diff
@@ -1342,6 +1342,45 @@
       }
     }
   },
+  "HaveIBeenPwned": {
+    "type": "observable",
+    "python_module": "haveibeenpwned.HaveIBeenPwned",
+    "description": "Check if an email address has been involved in a data breach",
+    "disabled": false,
+    "external_service": true,
+    "leaks_info": true,
+    "observable_supported": ["generic"],
+    "config": {
+      "soft_time_limit": 100,
+      "queue": "default"
+    },
+    "secrets": {
+      "api_key_name": {
+        "env_var_key": "HIBP_API_KEY",
+        "type": "string",
+        "required": true,
+        "default": null,
+        "description": "API Key for HaveIBeenPwned",
```

Comment:
the problem is this comma

Reply:
remove this and everything should be ok. JSON is very sensitive

Reply:
Thanks for the fix! I missed the comma out.